### DISCUSSION TITLE
Consider disabling autofill in certain areas

### DISCUSSION THREAD

**Discussion by matthewcare** [umbraco/Umbraco-CMS#13441] [Features and ideas]
In certain areas the autofill is more of a hinderance than useful.
This is an example of when I try to add a new link using the URL Picker.
The `Link` field is automatically focused, and I am faced with a list of random URLs which I have used.
Often times the autofill values are useless because they are from a completely different project.
![image](https://user-images.githubusercontent.com/1469061/202848737-8c65959b-af7c-4d1f-8d0a-59f62720319f.png)

What are the thoughts of other developers here?

**Comment by lassefredslund** [comment]
Hi @matthewcare. Thank you for sharing this. I think you are right that autofill is not always helpful. 
How do you see the best solution here? 

- We could fully disable autofill on all property editors (maybe based on a configuration) - but then you'll lose the opportunity to have it on individual property editors.
- We could also try to handle it for each specific property editor (e.g. with a toggle in the UI) - but then you'll see this toggle for all property editors.

And there could maybe be other/better suggestions. What do you think would work best?
